php/class-media.php

```diff
@@ -1549,7 +1549,8 @@ public function prepare_size( $attachment_id, $size ) {
 			}
 		} elseif ( is_string( $size ) || ( is_array( $size ) && 3 === count( $size ) ) ) {
 			$intermediate = image_get_intermediate_size( $attachment_id, $size );
-			if ( is_array( $intermediate ) ) {
+			// PDF's do not have intermediate URL.
+			if ( is_array( $intermediate ) && ! empty( $intermediate['url'] ) ) {
```

Comment:
<!-- potential concern, ask for clarification or investigation -->
**❓question**
You didn't mention this so I'm double checking it's supposed to be in here. Not a blocker!